vignettes/vignette_best_practices.Rmd

```diff
@@ -0,0 +1,198 @@
+---
+title: "Best practices when writing vignette in admiral"
```

Comment:
Change to "writing a vignette" or "writing vignettes"

Reply:
OK updated. I have also updated the file name.